package.json

```diff
@@ -4,7 +4,8 @@
   "description": "Convert JSON schema to Google BigQuery schema",
   "main": "src/index.js",
```

Comment:
Can you change this to `src/convertor.js` so the module can be required by other scripts.